code/game/objects/items/weapons/holosign.dm

```diff
@@ -31,23 +31,25 @@
 			if(!is_blocked_turf(T, TRUE)) //can't put holograms on a tile that has dense stuff
 				if(holocreator_busy)
 					to_chat(user, "<span class='notice'>[src] is busy creating a hologram.</span>")
-					return
+					return FALSE
 				if(signs.len < max_signs)
 					playsound(src.loc, 'sound/machines/click.ogg', 20, 1)
 					if(creation_time)
 						holocreator_busy = TRUE
 						if(!do_after(user, creation_time, target = target))
 							holocreator_busy = FALSE
-							return
+							return FALSE
 						holocreator_busy = FALSE
 						if(signs.len >= max_signs)
-							return
+							return FALSE
 						if(is_blocked_turf(T, TRUE)) //don't try to sneak dense stuff on our tile during the wait.
-							return
+							return FALSE
 					H = new holosign_type(get_turf(target), src)
 					to_chat(user, "<span class='notice'>You create [H] with [src].</span>")
+					return H
 				else
 					to_chat(user, "<span class='notice'>[src] is projecting at max capacity!</span>")
+					return FALSE
```

Comment:
If this is returning a type, dont denote the other return types as `FALSE`, leave them default (null). 

FALSE is not null